src/rebar_dialyzer_format.erl

```diff
@@ -79,6 +81,11 @@ fmt(Fmt) ->
 fmt(Fmt, Args) ->
     cf:format(Fmt, Args).
 
+line_from_line_col({Line,_Col}) ->
+    Line;
+line_from_line_col(Line) ->
+    Line.
+
```

Comment:
It might be interesting to eventually support displaying the column information, i.e. returning (either `io_lib:format("~w", [Line])` or `io_lib:format("~w (at column ~w)", [Line, Col])` and replacing the callsite to use `~ts` for the value. That way we show all the info we have if it's available.

Reply:
Actually, the original implementor wasn't completely aware there was a breaking change and is now contemplating adding an option like `{error_location, line | column}` to `dialyzer`'s. Shall we wait to see how it goes? We could probably piggy-back on that, then, and just pass the option along to `dialyzer`.

Reply:
Sure, it's fine to wait. We can see where this is headed but it'd be nice to give people the most information they can get and support columns when possible

Reply:
Ref: https://github.com/erlang/otp/pull/3018#issuecomment-772405965

Reply:
So, https://github.com/erlang/otp/pull/3018 was merged. I guess we just need to decide, now, on the actual format+phrasing for the output to include column.

Reply:
See my first comment for a possible formatting.

Reply:
Would it be acceptable to have an option to have line-only in the `rebar3`-generated output? Something like
```erlang
{dialyzer, [
    {error_location, line}
]}
```

I feel the output with the column would eat too much useful space, but it's your call, in the end.

Reply:
If you have a more compact idea that can work fine too. I don't know that people would really want to tweak this to save <10 characters of output, if we can get something nicer and keep it config free that'll be better.

Reply:
I can only think of the ol' `xx:yy` but that would generate it as e.g. `2474:13: The call user_st`, which is what `dialyzer` (the CLI tool) is doing with this new version. I'll move with what you proposed initially.

Reply:
Yeah sounds good.

Reply:
It's now producing something like
```erlang
src/mylib_app.erl
15 (at column 12): Unknown type my_type:is_unknown/0
```

I've also experimented with
```erlang
src/mylib_app.erl
Line 15, column 12: Unknown type my_type:is_unknown/0
```

and actually like it better (it's 1 char more than what I just pushed).

Reply:
Or even

```erlang
src/mylib_app.erl
Line 15 Column 12: Unknown type my_type:is_unknown/0
```

(same char. count) - don't mind the colors, of course.

Reply:
I'm going to propose a further commit for this, that can easily be reverted if you think it's not a better solution.

Reply:
f734736: this one prints mostly like is shown above: Line and Column are grey, 15 and 12 are cyan.

Reply:
Cyan is likely to look pretty bad on light background

Reply:
I didn't innovate there 😄. It was already _cyan_ (`~!c`). Want me to change it, though?

Reply:
eh if it was the same it's fine.